frontend/src/main/components/Commons/CommonsForm.js

```diff
@@ -24,11 +24,25 @@ function CommonsForm({initialCommons, submitAction, buttonLabel = "Create"}) {
 
   const testid = "CommonsForm";
 
+  // Default Values for CommonsForm fields
+  const curr = new Date();
+  const today = curr.toISOString().substring(0, 10);
+  //const onemonthfromtoday = new Date(curr.getFullYear(), curr.getMonth()+1, curr.getDate()).toISOString().substring(0, 10);
```

Comment:
What's this doing here?

Reply:
Is it for a future "end date" functionality? If so, then remove it until that new date is added.